src/components/Modal/Strategy.vue

```diff
@@ -39,6 +40,16 @@ function handleSubmit() {
   emit('close');
 }
 
+function handleClick(strategy) {
+  const { params } = strategy.examples[0].strategy;
+  const hasStrategy = !isEmpty(params);
+
+  input.value = {
+    name: strategy.key,
+    params: hasStrategy ? JSON.stringify(params, null, 2) : ''
```

Comment:
Could you use `defaultParams` as default value here
```suggestion
    params: hasStrategy ? JSON.stringify(params, null, 2) : defaultParams
```